.prettierrc.js

```diff
@@ -1,6 +1,10 @@
+/** @type {import('prettier').Config} */
 module.exports = {
   bracketSpacing: false,
   singleQuote: true,
   trailingComma: 'es5',
-  endOfLine: "auto"
-};
\ No newline at end of file
+  endOfLine: 'auto',
+  overrides: [
+    {files: '**/*.{scss,css,pcss,html,mdx}', options: {printWidth: 120}},
+  ],
```

Comment:
Not extremely related to the PR. I moved this from `package.json` so that we can format any file locally without committing.